snapcraft/internal/lxd/_project.py

```diff
@@ -44,23 +45,14 @@ def _ensure_container(self):
                 subprocess.check_call([
                     'lxc', 'init', self._image, self._container_name])
             except subprocess.CalledProcessError as e:
-                raise errors.ContainerConnectionError(
-                    'Failed to setup container')
+                raise errors.ContainerCreationFailedError() from e
         if self._get_container_status()['status'] == 'Stopped':
             self._configure_container()
             try:
                 subprocess.check_call([
                     'lxc', 'start', self._container_name])
             except subprocess.CalledProcessError:
-                msg = 'The container could not be started.'
-                if self._remote == 'local':
-                    msg += ('\nThe files /etc/subuid and /etc/subgid need to '
-                            'contain this line for mounting the local folder:'
-                            '\n    root:1000:1'
-                            '\nNote: Add the line to both files, do not '
-                            'remove any existing lines.'
-                            '\nRestart lxd after making this change.')
-                raise errors.ContainerConnectionError(msg)
+                raise errors.ContainerStartFailedError()
```

Comment:
This error discusses `/etc/subuid` etc, regardless of whether the remote is local or remote. Is that logic we want to keep? Because this changes how it was working previously.

Reply:
Support for remotes was removed so this is always local now.